src/bikesList.tsx

```diff
@@ -0,0 +1,222 @@
+        display: 'flex',
+        flexWrap: 'wrap',
+      },
+      formControl: {
+        margin: theme.spacing(1),
+        minWidth: 120,
+      },  
+    }),
+  );
+  const themeWarning = createMuiTheme({
+    palette: {
+      primary:{
+        main:'#950740'
+      }
+    },
+  });
+  const themeListItem = createMuiTheme({
+    palette: {
+      primary:{
+        main:'#4E4E50'
+      }
+    },
+  });
+  let bicycles: Bike[] = [];
+  // bicycles.push({ID: 10,State: BikeState.Blocked,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 12,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  // bicycles.push({ID: 11,State: BikeState.Working,StationID: 22});
+  
+
+  
+const  BikeListPage = () => {
+    const classes = useStyles();
+    const [openSlidingWindow, setOpenSlidingWindow] = useState<boolean>(false);
+    const [open, setOpen] = useState<boolean>(false);
+    const [newBikeState, setState] = React.useState<number>(0);
+    const [newBikeStationID, setStation] = React.useState<number>(0);
+    const [list, setList] = React.useState<Bike[]>(bicycles);
+    const [selectedIndex, setSelectedIndex] = React.useState(-1);
+    const handleListItemClick = (
+      index: number,
+    ) => {
+      setSelectedIndex(index);
+    };
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setState(Number(event.target.value));
+  };
+  const handleChangeStation = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setStation(Number(event.target.value));
+  };
+    const handleCloseSlidingWindow = () => {
+      setOpenSlidingWindow(false);
+    };
+    const deleteClicked = () => {
+      const newList = list.filter((item) => item.ID != list[selectedIndex].ID);  
+      setList(newList);
+      setOpenSlidingWindow(false);  
+    };
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+  
+    const handleClose = () => {
+      setOpen(false);
+    };
+    const handleAddBike = () => {
+      postBike({ID: Math.floor(Math.random() * 100), State: newBikeState, StationID: newBikeStationID}).then(r => {        
```

Comment:
przy postowaniu roweru nie potrzebujemy wysyłać `ID`, pogadamy jeszcze jak się to koszernie robi